test/fixtures/subtract.js

```diff
@@ -0,0 +1,13 @@
+const daysOfSun = 250,
+      foods = ['tzatziki','feta','gyros'],
+      islands = { inhabited: 170, total: 2000}
```

Comment:
'islands' is assigned a value but never used. A space is required before '}'. Missing semicolon.